src/test/java/fi/aalto/cs/apluscourses/model/ModelExtensions.java

```diff
@@ -71,13 +76,19 @@ public void updateState() {
     @Override
     public Course createCourse(@NotNull String name,
                                @NotNull List<Module> modules,
+                               @NotNull List<Library> libraries,
                                @NotNull Map<String, String> requiredPlugins) {
-      return new Course(name, modules, requiredPlugins);
+      return new Course(name, modules, libraries, requiredPlugins);
     }
 
     @Override
     public Module createModule(@NotNull String name, @NotNull URL url) {
       return new TestModule(name, url);
     }
+
+    @Override
+    public Library createLibrary(@NotNull String name) {
+      return null;
```

Comment:
https://github.com/Aalto-LeTech/intellij-plugin/blob/0df3dabd8ba864a7f5a6661ade669bf704448b68/src/main/java/fi/aalto/cs/apluscourses/intellij/model/IntelliJModelFactory.java#L76

Reply:
Good point! That would be better here too.